en/node-operators-guide/request-response.md

```diff
@@ -90,14 +90,55 @@ Errors and warnings emitted by the **Orakl Network Request-Response** can be [se
 
-After the **Orakl Network API** is healthy, Request-Response microservices (listener, worker, reporter) can be launched in an arbitrary order. Microservices communicate with each other through the BullMQ - job queue.
+After the **Orakl Network API** is healthy, launch the VRF service, which consists of listener, worker, and reporter microservices, with the command below. Microservices communicate with each other through the BullMQ - job queue.
+
+```sh
+yarn start:core:request_response
+```
+
+For development purposes, the service can be run with `nodemon`, which automatically restarts all the microservices when file changes are detected, using the following command:
+
+```sh
+yarn dev:core:request_response
+```
+
+It's also possible to run the microservices separately in any arbitrary order:
 
 ```sh
 yarn start:listener:request_response
 yarn start:worker:request_response
 yarn start:reporter:request_response
 ```
 
+## Quick launch with Docker
+
+From [orakl]() repository's root, run the following command to build all images:
+
+```bash
+docker-compose -f docker-compose.local-core.yaml build
+```
+
+Set wallet credentials, `ADDRESS` and `PRIVATE_KEY` values, in the [.core-cli-contracts.env](https://github.com/Bisonai/orakl/blob/master/dockerfiles/local-vrf-rr/envs/.core-cli-contracts.env) file. Keep in mind that the default chain is `localhost`. If you'd like to change it to either `baobab` or `cypress`, update `CHAIN` and `PROVIDER_URL` values in the earlier mentioned `.env` file. Note that if the chain is not `localhost`, `Coordinator` and `Prepayment` contracts won't be deployed. Instead, Bisonai's already deployed contract addresses ([VRF](https://github.com/Bisonai/vrf-consumer/blob/master/hardhat.config.ts), [RR](https://github.com/Bisonai/request-response-consumer/blob/master/hardhat.config.ts)) will be used. After setting the appropriate `.env` values, run the following command to start the Request-Response service:
+
+```bash
+SERVICE=rr docker-compose -f docker-compose.local-core.yaml up --force-recreate
+```
+
+**Note** that the current docker implementation is designed to run a single service, either `rr` or `vrf`, at a time. Therefore, it's highly recommended to add `--force-recreate` when running `docker-compose up` command. That will restart all containers thus removing all the modified data in those containers.
+
+Here is what happens after the above command is run:
+
+- `api`, `postgres`, `redis`, and `json-rpc` services will start as separate docker containers
+- `postgres` will get populated with necessary data:
+  - chains
+  - services
+  - listener (after contracts are deployed)
+  - reporter (after contracts are deployed)
+- migration files in `contracts/v0.1/migration/` get updated with provided keys and other values
+- if the chain is `localhost`:
+  - `contracts/v0.1/hardhat.config.cjs` file gets updated with `PROVIDER_URL`
+  - relevant coordinator and prepayment contracts get deployed
```

Comment:
how about using numbered list here if it is ordered one

Reply:
it might be a bit hard to number according to the execution order because some of these run in parallel and others wait for certain conditions. And there are conditional cases as well